pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/bloom/BloomIndexType.java

```diff
@@ -70,9 +70,14 @@ public BloomFilterConfig getDefaultConfig() {
     return BloomFilterConfig.DISABLED;
   }
 
+  @Override
+  public String getPrettyName() {
+    return "bloom";
```

Comment:
Can we please extract this name as a constant before?

Reply:
Will do.